src/main/resources/se/vandmo/dependencylock/maven/pom/pom-with-plugins.ftlx

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="
+    http://maven.apache.org/POM/4.0.0 https://vandmo.github.io/dependency-lock-maven-plugin/maven-v4_0_0_ext.xsd
+    urn:se.vandmo.dependencylock https://vandmo.github.io/dependency-lock-maven-plugin/dependencylock.xsd"
+  xmlns:dependency-lock="urn:se.vandmo.dependencylock">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>${pom.groupId}</groupId>
+  <artifactId>${pom.artifactId}-dependency-lock</artifactId>
+  <version>${pom.version}</version>
+
+  <dependencies>
+    <#list dependencies as dependency>
+    <dependency>
+      <groupId>${dependency.artifactIdentifier.groupId}</groupId>
+      <artifactId>${dependency.artifactIdentifier.artifactId}</artifactId>
+      <version>${dependency.version}</version>
+      <type>${dependency.artifactIdentifier.type}</type>
+      <scope>${dependency.scope}</scope>
+      <#if dependency.artifactIdentifier.classifier.isPresent()>
+      <classifier>${dependency.artifactIdentifier.classifier.get()}</classifier>
+      </#if>
+      <optional>${dependency.optional?c}</optional>
+      <dependency-lock:integrity>${dependency.getIntegrityForLockFile()}</dependency-lock:integrity>
+    </dependency>
+    </#list>
+  </dependencies>
+
+  <build>
```

Comment:
Inconsistent indentation, should be 2 spaces everywhere

Reply:
Fixed in https://github.com/vandmo/dependency-lock-maven-plugin/pull/85/commits/ad1c38b4377abb9d4fe8d820a78b7a5b0927c538